x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import datemath from '@elastic/datemath';
+import expect from '@kbn/expect';
+import mockRolledUpData, { mockIndices } from './hybrid_index_helper';
+
+export default function ({ getService, getPageObjects }) {
+
+  const es = getService('es');
+  const esArchiver = getService('esArchiver');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common', 'settings']);
+
+  describe('hybrid index pattern', function () {
+    const rollupJobName = `hybrid-index-pattern-test-rollup-job-` + Math.floor(Math.random() * 10000);
+    const rollupTargetIndexName = `rollup-target-data`;
+    const regularIndexPrefix = `regular-index`;
+    const rollupSourceIndexPrefix = `rollup-source-data`;
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
+
+    it('create hybrid index pattern', async () => {
+      //Create data for rollup job to recognize.
+      //Index past data to be used in the test.
+      await pastDates.map(async (day) => {
+        await es.index(mockIndices(day, rollupSourceIndexPrefix));
+      });
+
+      await retry.waitForWithTimeout('waiting for 3 records to be loaded into elasticsearch.', 10000, async () => {
+        const response = await es.indices.get({
+          index: `rollup-source-data*`,
```

Comment:
Can we use the variable here instead of hardcoding `rollup-source-data`?

Reply:
Yep. Fixed.